lib/active_merchant/billing/gateways/quickpay/quickpay_v10.rb

```diff
@@ -15,7 +15,7 @@ def initialize(options = {})
       end
 
       def purchase(money, credit_card_or_reference, options = {})
-        MultiResponse.run(true) do |r|
+        process = MultiResponse.run(true) do |r|
```

Comment:
Indeed that works. I was thinking that it was set to use the first response for error/message reporting purposes, but removing it doesn't cause any negative remote test failures, at least.